bundles/CoreBundle/EventListener/Frontend/DocumentFallbackListener.php

```diff
@@ -120,6 +120,7 @@ public function onKernelRequest(RequestEvent $event)
                 if ($document->getProperty('language')) {
                     $request->setLocale($document->getProperty('language'));
                 }
+                $this->documentResolver->setDocument($event->getRequest(), $this->fallbackDocument);
```

Comment:
@ckogler-elements this seems to be an unrelated change - can you fix that? Thanks!